src/main/java/com/example/playdocker/services/MessageService.java

```diff
@@ -0,0 +1,198 @@
+package com.example.playdocker.services;
+
+import com.example.playdocker.Utils.MessageUtils;
+import com.example.playdocker.models.Message;
+import com.example.playdocker.models.Score;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class MessageService {
+    public static boolean subscribe(String senderNumber) {//Funkcja odpowiedzialna za zapisanie numeru telefonu do pliku CSV jeśli numer nie występuje jeszcze w bazie.
+        String environment = MessageUtils.getEnvironmentFromProperties();//Wyciągnięcie zmiennej mówiącej czy operujemy na środowisku testowym czy produkcyjnym.
+        String filePath;//Plik pełniący funkcję bazy.
+        boolean isNumberInDatabase;
+        boolean isSubscriptionOk = false;
+
+        if(environment.equalsIgnoreCase("produkcja")) {//Zależnie od środowiska wybierane jest na której "bazie" operować
+            filePath = "numeryTelefonówProd.csv";
+        } else {
+            filePath = "numeryTelefonówTest.csv";
+        }
+
+        isNumberInDatabase = checkIsNumberInBase(senderNumber, filePath);//Sprawdzenie czy numeru nie ma już w bazie
+
+        if (isNumberInDatabase) {//Jeśli numer jest już w bazie, ustaw negatywną odpowiedź
+            isSubscriptionOk = false;
+        } else {//Jeśli numeru nie ma w bazie, jest on dodawany
+            try {
+                FileWriter csvWriter = new FileWriter(filePath, true);
+
+                csvWriter.append(senderNumber);
+                csvWriter.append(",");
+                csvWriter.append("\n");
+
+                csvWriter.flush();
+                csvWriter.close();
+                isSubscriptionOk = true;
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+        }
+        return isSubscriptionOk;
+    }
+
+    public static boolean unsubscribe(String numberToRemove) {//Funkcja odpowiedzialna za usunięcie numeru telefonu z pliku CSV jeśli podany numer w niej istnieje.
+        String environment = MessageUtils.getEnvironmentFromProperties();//Wyciągnięcie zmiennej mówiącej czy operujemy na środowisku testowym czy produkcyjnym.
+        String filePath;//Plik pełniący funkcję bazy.
+        boolean isNumberInDatabase;
+        boolean isSubscriptionOk = false;
+
+        if(environment.equalsIgnoreCase("produkcja")) {//Zależnie od środowiska wybierane jest na której "bazie" operować
+            filePath = "numeryTelefonówProd.csv";
+        } else {
+            filePath = "numeryTelefonówTest.csv";
+        }
```

Comment:
Abstrahując już od tego, że ten pomysł z CSV jest do przemyślenia :) to takie rzeczy możesz trzymać w konfiguracji - poczytaj o profilach w springu

Reply:
Obiecuję, że poczytam i na przyszłość się nauczę :)